5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
The `$OPTx` in this image are used to get user/pass/dbname from user, which are required to create a container. I don't have any other way to get those data except `$OPTx`, do I?
@hanzz not sure I got your crazy idea correctly, but when I tried something like `atomic install --unknown ...`, then I got `/bin/atomic: unrecognized arguments: --unknown`...